### ISSUE TITLE
W1302 Invalid format string triggered by implicitly-joined literal string

### ISSUE THREAD

**Issue by pylint-bot** [PyCQA/pylint#348]
Originally reported by: **BitBucket: [dmnd](http://bitbucket.org/dmnd), GitHub: @dmnd?**

---

This looks like a false positive to me, but I could be wrong. "W1302 Invalid format string" is triggered on the following code:

```
t = 'bad'
s = (u"Implicitly joined {!r}} "
      "string".format(t))
```

---
- Bitbucket: https://bitbucket.org/logilab/pylint/issue/348
